doc/Caliptra.md

```diff
@@ -61,6 +61,8 @@ The Caliptra Workgroup acknowledges the following individuals for their contribu
 |January 2023|0.61|Jeff Andersen|Removed KEY MANIFEST SVN from spec.|
 |February 2023|0.62|Piotr Kwidzinski|Moved spec to Caliptra GitHub repo.|
 |May 2023|0.63|Caleb Whitehead|Updated Error Reporting and Handling HW Error Table.|
+|May 2023|0.7|Piotr Kwidzinski|Cleanup references, links and figures. Minor edits|
```

Comment:
This was needed when the specification was a word document, but now that it's in Git is it still needed?

Reply:
perhaps not but will leave it for now

Reply:
Yeah there is a separate effort that @steph-morton is leading so we'll fix it there.